### PR TITLE
Display remaining time to make changes to affect next FC

### LINKED ISSUE #185: Show how much time remains for making a change to the next funding cycle, based on ballot
**Is your feature request related to a problem?**
If a delay ballot is being used, it can be difficult to figure out how much time is left to reconfigure the upcoming funding cycle.

**Describe the feature/solution you'd like**
Based on any ballot delay, the UI should make it clear how much more time remains to submit a funding cycle reconfiguration that will take effect once the current cycle ends.

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Closes #185. Displays how much time a user has to make changes which will affect the upcoming FC. Working on V1 and V2 projects, had to add 'durationSeconds' property to the V1 ballot parsing.

Open to copy improvements or anything you think would make the information more clear.

## Screenshots or screen recordings

Screen Shot 2022-07-25 at 4 28 35 PM

V2 👆

Screen Shot 2022-07-25 at 4 53 36 PM

V1 👆

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/constants/v1/ballotStrategies/index.ts [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 
+import { SECONDS_IN_DAY } from 'constants/numbers'
+
 export function ballotStrategies(): BallotStrategy[] {
   return [
     {
       name: t`No strategy`,
       description: t`Any reconfiguration to an upcoming funding cycle will take effect once the current cycle ends. A project with no strategy may be vulnerable to being rug-pulled by its owner.`,
       address: constants.AddressZero,
+      durationSeconds: 0,
```
Comment (peripheralist) [inline src/constants/v1/ballotStrategies/index.ts:14]: nice! should we also add `durationSeconds` to the `BallotStrategy` type?

File: src/components/v1/V1Project/modals/ReconfigureFCModal.tsx [hunk 2]
```diff
@@ -59,8 +59,86 @@ import TicketingForm, {
...
+  currentFC,
+}: {
+  currentFC: V1FundingCycle | undefined
+}) => {
+  if (!currentFC) return null
+  const currentFCNumber = currentFC.number.toNumber()
+  const ballotStrategy = getBallotStrategyByAddress(currentFC.ballot)
+  const ballotStrategyLength = ballotStrategy.durationSeconds
+  const duration = currentFC.duration.toNumber()
+  const durationInSeconds = duration * 24 * 60 * 60
```
Comment (peripheralist) [inline src/components/v1/V1Project/modals/ReconfigureFCModal.tsx:76]: could use `duration * SECONDS_IN_DAY` here

**Review by peripheralist — COMMENTED** [review]
love this @jmilldotdev ! real clean. just a few small comments

File: src/constants/v1/ballotStrategies/index.ts (hunk 1 above)
Comment (jmilldotdev) [reply src/constants/v1/ballotStrategies/index.ts:14]: it is on the type already as an optional

Comment (peripheralist) [reply src/constants/v1/ballotStrategies/index.ts:14]: 🤦

**Review by peripheralist — APPROVED** [review]

**Pull request merged**